<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Declaration</class>
 <widget class="QDialog" name="Declaration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>545</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Déclaration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: #e6ebf4;
}
/*----------------- Objets----------------- */
QLineEdit{
border: 1px solid gray;
border-style: solid;
border-radius: 2px;
padding: 0 4px;
background: white;
selection-background-color: darkgray;
}
QSpinBox{
border: 1px solid gray;
border-style: solid;
border-radius: 2px;
padding: 0 4px;
background: white;
selection-background-color: darkgray;
alignment: alignright;
}
QDateEdit{
border: 1px solid gray;
border-style: solid;
border-radius: 2px;
padding: 0 4px;
background: white;
selection-background-color: darkgray;
}

/*-----------------Textes----------------- */
/*  Titres */
QLabel.titreA{
color: rgb(166, 54, 110);
font: Arial;
font-size : 20px;
font-weight : normal;
border-style:solid;
border-color:white;
border-width:5px;
}
QLabel.titreB{
color: rgb(166, 54, 110);
font: Arial;
font-size : 14px;
font-weight : normal;
text-decoration:underline
}
QLabel.titreC{
color: rgb(166, 54, 110);
font: Arial;
font-size : 12px;
font-weight : normal;
}
QLabel.titreD{
color: rgb(166, 54, 110);
font: Arial;
font-size : 9px;
font-weight : normal;
}

/*----------------- Design----------------- */
/*------  page  declaration.ui------*/

/*  boite générale */
QLabel.sponsor{
background-image: url(:/images/logo1.png);  
}

QFrame.top{
background-color: #e7d2d9;
border: 2px solid #e7d2d9;
border-radius: 6px;
}

/*  Pages du formulaire */
QLabel.boite1{
background-color: #e7d2d9;
border-top: 2px solid #e7d2d9;
border-left: 2px solid #e7d2d9;
border-right: 2px solid #e7d2d9;
border-top-left-radius: 6px;
border-top-right-radius: 6px;
font-size: 20px;
text-align:center;
font: bold;
color: #ffffff;
}
QListView {
border-left: 2px solid #e7d2d9;
border-right: 2px solid #e7d2d9;
border-bottom: 2px solid #e7d2d9;
font-size: 10px;
border-bottom-left-radius: 6px;
border-bottom-right-radius: 6px;
}
/*  formulaire */
QScrollArea QWidget{
background-color: white;
}

QLabel.texte01{
font-size: 11px;
}
QLabel.texte02{
font-size: 9px;
}
QLabel.code{
font-size: 12px;
font-weight : bold;
}
</string>
  </property>
  <property name="locale">
   <locale language="French" country="France"/>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>27</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>PAGES</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="class" stdset="0">
        <string>boite1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="contents_widget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>108</width>
           <height>210</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>108</width>
           <height>210</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="class" stdset="0">
          <string>sponsor</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>27</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="class" stdset="0">
        <string>top</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>12</number>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Précédent</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="prev_btn">
            <property name="text">
             <string>...</string>
            </property>
            <property name="arrowType">
             <enum>Qt::LeftArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>27</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>FORMULAIRE</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="class" stdset="0">
             <string>boite1</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="next_btn">
            <property name="text">
             <string>...</string>
            </property>
            <property name="arrowType">
             <enum>Qt::RightArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Suivant</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>749</width>
             <height>490</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Declaration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Declaration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
